api-microservice/.eslintrc

```diff
@@ -4,10 +4,15 @@
     "project": "tsconfig.json",
     "sourceType": "module"
   },
-  "plugins": ["@typescript-eslint"],
+  "plugins": ["@typescript-eslint", "jest", "prettier"],
   "extends": ["airbnb-typescript/base", "prettier", "prettier/@typescript-eslint"],
   "root": true,
+  "ignorePatterns": ["node_modules/**", "dist/**"],
   "rules": {
-    "quotes": ["error", "single"]
+    "quotes": ["error", "single"],
```

Comment:
This line can be deleted as the `prettier` plugin gets it from the `.prettierrc` file

Reply:
Done.